plastron/util.py

```diff
@@ -25,6 +25,10 @@
 logger = logging.getLogger(__name__)
 
 
+def is_handle(value: str) -> bool:
```

Comment:
This function should go in the `plastron.validation` module instead, alongside `is_edtf_formatted` and `is_valid_iso639_code`.